differential/customfield/DifferentialBugzillaBugIDField.php

```diff
@@ -82,11 +107,10 @@ public function shouldAppearInPropertyView() {
   }
 
   public function renderPropertyViewValue(array $handles) {
-    // TODO Link to the right Bugzilla server (defined in docker)
-    return phutil_tag(
-      'a', array('href' => 'https://bugzilla/'.$this->getValue()),
-      $this->getValue()
-    );
+    $bug_uri = (string) id(new PhutilURI(PhabricatorEnv::getEnvConfig('bugzilla.url')))
+      ->setPath($this->getValue());
```

Comment:
This produces a bug url like `http://bmo.test/3`, but, it needs to look like `http://bmo.test/show_bug.cgi?id=3`